libs/vm-modules/benchmark/scripts/process_bench_output.py

```diff
@@ -0,0 +1,104 @@
+import os
```

Comment:
i don't think python files are supposed to live in /libs generally. Presumably this can move to scripts.

Reply:
sure, we'll confirm that with Ed